lib/workers/repository/extract/index.ts

```diff
@@ -47,6 +47,16 @@ export async function extractAllDependencies(
   const extractResults = await Promise.all(
     extractList.map(async (managerConfig) => {
       const packageFiles = await getManagerPackageFiles(managerConfig);
+      for (const p of packageFiles) {
+        //istanbul ignore if
```

Comment:
why we don't have added a test here? 😕

Reply:
I think this actually broke 32.66.6 for us:

```
{
           "name": "renovate",
           "level": 50,
           "logContext": "VeQ76084oIBuT1WliExx-",
           "repository": "intern/symfony-oidc",
           "err": {
             "message": "packageFiles is not iterable",
             "stack": "TypeError: packageFiles is not iterable\n    at /usr/src/app/node_modules/renovate/lib/workers/repository/extract/index.ts:50:23\n    at async Promise.all (index 1)\n    at extractAllDependencies (/usr/src/app/node_modules/renovate/lib/workers/repository/extract/index.ts:47:26)\n    at extract (/usr/src/app/node_modules/renovate/lib/workers/repository/process/extract-update.ts:82:20)\n    at extractDependencies (/usr/src/app/node_modules/renovate/lib/workers/repository/process/index.ts:105:26)\n    at Object.renovateRepository (/usr/src/app/node_modules/renovate/lib/workers/repository/index.ts:42:52)\n    at Object.start (/usr/src/app/node_modules/renovate/lib/workers/global/index.ts:150:7)\n    at /usr/src/app/node_modules/renovate/lib/renovate.ts:16:22"
           },
           "msg": "Repository has unknown error"
         },
```

Reply:
`packageFiles` can be null. 

@rarkins please revert this PR, to start again with tests.